### ISSUE TITLE
Privacy: sentry is still sending some data when analytics is off

### ISSUE THREAD

**Issue by leohearts** [tooot-app/app#475]
Version: 4.6.1
Steps to reproduce:
- turn off analytics
- start monitoring network requests
- restart tooot

![image](https://user-images.githubusercontent.com/24632029/202914118-056fb597-63b6-4f7a-9c08-1f587769b854.png)
![image](https://user-images.githubusercontent.com/24632029/202914123-66750411-0bd9-4ebf-901c-168d39581ff9.png)

Is that because we only turns off firebase ? https://github.com/tooot-app/app/blob/dde33ad1ad48ea4319af63fe5167a5fc90317026/src/utils/slices/settingsSlice.ts#L9-L15

**Comment by xmflsct** [comment]
> Is that because we only turns off firebase ?

Yes. Firebase Analytics is none essential for app performance, therefore users can choose to opt out. It was added since the very beginning with the intention to be able to analyse behavioural data to improve interaction design. However I never had the time to actually analyse anything, and actions added later are not tracked at all.

At some point I may just remove Firebase Analytics all together. Though Sentry is essential to understand app's performance, which also is listed in the [privacy policy](https://tooot.app/privacy-policy).
